modules/imaging_browser/php/NDB_Form_imaging_browser.class.inc

```diff
@@ -51,7 +51,7 @@ class NDB_Form_Imaging_Browser extends NDB_Form
         // allow only to view own site data
         $timePoint =& TimePoint::singleton($_REQUEST['sessionID']);
 
-        return ($user->hasPermission('imaging_browser_view_allsites') || ((in_array($timePoint->getData('CenterID'), $user->getData('CenterIDs'))) && $user->hasPermission('imaging_browser_view_site')));
+        return ($user->hasPermission('imaging_browser_view_allsites') || ((in_array($timePoint->getData('CenterID'), $user->getData('CenterIDs'))) && $user->hasPermission('imaging_browser_view_site')) || $user->hasPermission('imaging_browser_phantom_data'));
```

Comment:
@driusan 
Imaging browser files are not part of PHPCS on Travis (the whole code has these very long lines).
I can fix my newly edited lines if you prefer,
but having PHPCS run on the whole module is what I believe we should do in a separate pull request (which I believe @kongtiaowang is doing)